File: Python/Fundamentals/Functions/Exercise_Functions/odd_and_even_sum.py
def sum_of_digits(digit):
    sum_even = 0
    sum_odd = 0
    for digit in str(number):
        if int(digit) % 2 == 0:
            sum_even += int(digit)
        else:
            sum_odd += int(digit)

    return f"Odd sum = {sum_odd}, Even sum = {sum_even}"

number = int(input())
print(sum_of_digits(number))

File: Python/JS-Front-End/Functions-and-statements/Exercise/odd-and-even-sum.js
function solve(number) {

    let numberAsString = number.toString();
    let even = 0;
    let odd = 0;

    for (const digit of numberAsString) {
        currentDigit = Number(digit)

        if (currentDigit % 2 === 0) {
            even += currentDigit
        } else {
            odd += currentDigit
        }

    } 

    console.log(`Odd sum = ${odd}, Even sum = ${even}`)
}

solve(1000435)